sympy/solvers/polysys.py

```diff
@@ -429,3 +439,389 @@ def solve_triangulated(polys, *gens, **args):
 
         solutions = _solutions
     return sorted((s for s, _ in solutions), key=default_sort_key)
+
+
+def factor_poly_system(eqs, *gens, **kwargs):
+    """
+    Given a system of polynomial equations, returns a factored form and solvability conditions.
+
+    Parameters
+    ==========
+
+    eqs : list
+        List of expressions to be factored. Each expression is assumed to be equal to zero.
```

Comment:
Prose text in docstrings should be wrapped at 78 characters. Generally lines of code should not be much longer than that but particularly prose text should always be wrapped.